src/ServerRequestFromGlobalsFactoryInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Psr\Http\Factory;
+
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Message\UriInterface;
+
+interface ServerRequestFromGlobalsFactoryInterface
```

Comment:
I thought about that, but it breaks grouping by filename and doesn't match the other interfaces/method naming.